src/components/progress/progress.scss

```diff
@@ -0,0 +1,114 @@
+@import '../../themes/ionic.globals';
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:rgb(16,108,200) !default;
```

Comment:
Something lint won't catch, is variables alignment. Please keep all variable values aligned at the same column